tests/functional_tests/t0_main/lightgbm/t2_regression.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+"""Test lightgbm integration."""
+
+import lightgbm as lgb
+import pandas as pd
+import requests
+import wandb
+from wandb.integration.lightgbm import log_summary, wandb_callback
+
+print(lgb.__version__)
```

Comment:
oh you want to keep it as part of the test... i don't mind either way

Reply:
Yeah I wanted to keep it as part of the test.